chart/linkerd-buoyant/templates/agent.yaml

```diff
@@ -0,0 +1,438 @@
+#
+# Secrets to identify and authenticate this agent
+#
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: buoyant-cloud-id
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/part-of: buoyant-cloud
+type: Opaque
+data:
+  id: {{ b64enc (required "Please provide the ID for your Buoyant Cloud account in the accountID value" .Values.accountID | trim) }}
+  key: {{ b64enc (required "Please provide the key for your Buoyant Cloud account in the accountKey value" .Values.accountKey | trim) }}
+  downloadKey: {{ b64enc (required "Please provide the download key for your Buoyant Cloud account in the accountDownloadKey value" .Values.accountDownloadKey | trim) }}
+  name: {{ b64enc (required "Please provide the cluster named the clusterName value" .Values.clusterName | trim) }}
+---
+#
+# Buoyant Cloud Agent
+#
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: buoyant-cloud-agent
+  namespace: {{ .Values.namespace }}
+  annotations:
+    buoyant.cloud/is-agent: "true"
+    buoyant.cloud/version: v0.2.0
+    buoyant.cloud/service-name: agent
+  labels:
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: v0.2.0
+    app.kubernetes.io/part-of: buoyant-cloud
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: buoyant-cloud-agent
+  template:
+    metadata:
+      labels:
+        app: buoyant-cloud-agent
+      annotations:
+        config.linkerd.io/skip-outbound-ports: "4191"
+    spec:
+      securityContext:
+        fsGroup: 1000
+      serviceAccount: buoyant-cloud-agent
+      containers:
+      - name: buoyant-cloud-agent
+        image: ghcr.io/buoyantio/linkerd-buoyant:v0.2.0
+        imagePullPolicy: Always
+        args:
+        - "-grpc-addr=api.staging-buoyant.cloud:443"
```

Comment:
I am pretty sure this needs to be configurable and the default should probably point to prod. Also, we need to be able to change that for local developement to point to something like `dockerhost:8086`

Reply:
:+1: